python/instrumentation/openinference-instrumentation-mistralai/src/openinference/instrumentation/mistralai/_chat_wrapper.py

```diff
@@ -87,17 +95,28 @@ def _get_attributes_from_request(
     ) -> Iterator[Tuple[str, AttributeValue]]:
         yield SpanAttributes.OPENINFERENCE_SPAN_KIND, self._get_span_kind()
         try:
-            yield SpanAttributes.INPUT_MIME_TYPE, OpenInferenceMimeTypeValues.JSON.value
-            yield SpanAttributes.INPUT_VALUE, json.dumps(request_parameters)
-            invocation_parameters = {
-                param_key: param_value
-                for param_key, param_value in request_parameters.items()
-                if param_key != "messages"
-            }
-            yield SpanAttributes.LLM_INVOCATION_PARAMETERS, json.dumps(invocation_parameters)
-            yield SpanAttributes.LLM_INPUT_MESSAGES, json.dumps(request_parameters.get("messages"))
+            yield from _as_input_attributes(_io_value_and_type(request_parameters))
+        except Exception:
+            logger.exception(
+                f"Failed to get input attributes from request parameters of "
+                f"type {type(request_parameters)}"
+            )
+
+    def _get_extra_attributes_from_request(
+        self,
+        request_parameters: Mapping[str, Any],
+    ) -> Iterator[Tuple[str, AttributeValue]]:
+        # Secondary attributes should be added after input and output to ensure
```

Comment:
This is actually not true, attributes fall off the top. There's an issue to track this

Reply:
So this should actually be reversed?

Reply:
addressed in 03f227a1e38e35a10b002e8bd06da1a59801ef5b

Reply:
related to #244

Reply:
Thanks!